src/quality_result_gui/ui/quality_error_tree_view.py

```diff
@@ -91,6 +94,16 @@ def mousePressEvent(  # noqa: N802 (override qt method)
 
         self.source_button_event = None
 
+    def get_all_quality_errors(self) -> List[QualityError]:
+        errors: List[QualityError] = []
+        for i in range(len(QualityErrorPriority)):
+            errors += list(
+                self._get_quality_errors_from_index(
```

Comment:
this could be probably simpler to understand with a list comprehension instead if iteration is necessary here?

`[error for i, _ in enumerate(QualityErrorPriority) for error in self._get_quality_errors_from_index(self.model.index(i, 0))]`